acceptance/repository-cloudant/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@loopback/test-repository-cloudant",
+  "version": "0.4.8",
+  "description": "",
+  "private": true,
+  "engines": {
+    "node": ">=8.9"
+  },
+  "scripts": {
+    "build": "lb-tsc",
+    "clean": "lb-clean loopback-test-repository-cloudant*.tgz dist tsconfig.build.tsbuildinfo package",
+    "pretest": "npm run build",
+    "test": "npm run mocha",
+    "mocha": "lb-mocha \"dist/__tests__/**/*.js\"",
+    "verify": "npm pack && tar xf loopback-test-repository-cloudant*.tgz && tree package && npm run clean"
+  },
+  "author": "IBM Corp.",
+  "copyright.owner": "IBM Corp.",
+  "license": "MIT",
+  "devDependencies": {
+    "@loopback/build": "^1.7.1",
+    "@loopback/eslint-config": "^4.1.2",
+    "@loopback/repository": "^1.15.2",
+    "@loopback/repository-tests": "^0.5.2",
```

Comment:
The version inside this repo is currently `0.6.1` which is not matched by `^0.5.2` so when `npm run build -- --scope "@loopback/test-repository-cloudant" --include-dependencies` is run it doesn't detect that it should be building `packages/test-repositories` and then when it goes to run the tests using that package, it has no `./dist` directory.

Reply:
Yes! it works once I updated it. Thank you so much!!